prow/getting_started_deploy.md

```diff
@@ -308,19 +313,25 @@ gcloud iam service-accounts create prow-gcs-publisher # step 1
 identifier="$(  gcloud iam service-accounts list --filter 'name:prow-gcs-publisher' --format 'value(email)' )"
 gsutil mb gs://prow-artifacts/ # step 2
 gsutil iam ch allUsers:objectViewer gs://prow-artifacts # step 3
-gsutil iam ch "serviceAccount:${identifier}:objectCreator" gs://prow-artifacts # step 4
+gsutil iam ch "serviceAccount:${identifier}:objectAdmin" gs://prow-artifacts # step 4
```

Comment:
`initupload` (or technically, `gcsupload` as invoked by `initupload`) wants to overwrite `build-latest.txt` and needs this to do so

Reply:
Hmm I wonder if this is safe since this is giving permission to jobs in the build cluster.
WDYT @stevekuznetsov?

Reply:
That was my thought as well, but based on `gsutil iam get gs://kubernetes-jenkins` it seemed like that's the route I needed to go; I don't have access to know exactly which one of those corresponds to the service account used by prow.k8s.io

Reply:
I don't think this was something i found in my testing?

Reply:
WIll have to check but our SA is not an admin IIRC

Reply:
![Screenshot from 2019-06-25 15-25-50](https://user-images.githubusercontent.com/7328370/60137993-993a5380-975d-11e9-96ea-d646711ece0e.png)

:man_facepalming:

Reply:
I am fine if we decide we need to do better, I agree it sounds scary, but it seems like I've documented the state of today.

Reply:
Agreed